src/content/contribute/writing-a-loader.md

```diff
@@ -361,13 +365,16 @@ And now, finally, we can write our test and add an npm script to run it:
 __test/loader.test.js__
 
 ```js
+/**
+ * @jest-environment node
+ */
 import compiler from './compiler.js';
 
 test('Inserts name and outputs JavaScript', async () => {
   const stats = await compiler('example.txt');
   const output = stats.toJson().modules[0].source;
 
-  expect(output).toBe('export default "Hey Alice!\\n"');
+  expect(output).toBe('export default "Hey Alice!\"');
```

Comment:
The `example.txt` should be:

```
Hey [name]!

```
Unfortunately, PrismJS would remove the line break, so I just remove the `\n` here.